libopae/plugins/xfpga/sysfs.c

```diff
@@ -108,11 +108,14 @@ STATIC int parse_pcie_info(sysfs_fpga_region *region, char *buffer)
 		FPGA_ERR("Error executing regex: %s", err);
 		res = FPGA_EXCEPTION;
```

Comment:
res is overwritten with FPGA_OK on line 116.  Probably need a goto out; here.

Reply:
Oops. You're right